### ISSUE TITLE
Improper shorthand compacting when !important is involved

### ISSUE THREAD

**Issue by hamilr2** [jakubpawlowicz/clean-css#795]
Hello, ran into this today using 3.4.18:

```
.foo {
  padding: 0 !important;
  padding-left: 3px;
}
```

becomes

```
.foo{padding:0 0 0 3px!important;}
```

Granted, the original CSS isn't really a use case and clearly doesn't work as intended (and has since been corrected!), but I was still surprised that the output had a significantly different meaning.

**Comment by jakubpawlowicz** [comment]
Thanks @hamilr2, much appreciated bug report! There was a bug in test suite which didn't catch it earlier.

**Comment by jakubpawlowicz** [comment]
It's fixed in 3.4.19.
